ppr-ui/src/components/mhrRegistration/YourHome/OtherInformation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-card flat class="white pb-6 pt-6 pr-10 pl-8 rounded">
+      <v-row>
+        <v-col cols="2">
+          <label class="generic-label ml" for="">Other</label>
+        </v-col>
+        <v-col cols="10">
+          <v-textarea
+            v-model="otherRemarks"
+            filled
+            :error-messages="errorMessages.otherRemarks"
+            name="name"
+            counter="140"
+            placeholder="Enter other details about the home (Optional)"
+            class="other-info"
+            data-test-id="otherRemarks"
+          ></v-textarea>
+        </v-col>
+      </v-row>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useActions, useGetters } from 'vuex-composition-helpers'
+import { useMhrRegistrationValidation as getMhrRegistrationErrors } from '@/composables/useMhrRegistrationValidation'
+
+export default defineComponent({
+  setup () {
+    const { getMhrRegistrationOtherInfo } = useGetters<any>([
+      'getMhrRegistrationOtherInfo'
+    ])
+
+    const { setMhrRegistrationOtherInfo } = useActions<any>([
+      'setMhrRegistrationOtherInfo'
+    ])
+
+    const localState = reactive({
+      otherRemarks: getMhrRegistrationOtherInfo.value,
+      errorMessages: {
+        otherRemarks: ''
+      }
```

Comment:
I'm not sure we need a local model of the error message when the component can handle this internally.
We would need a local model for determining validity however. 

Wrapping our input fields in a `v-form` is very useful for a few reasons, we can utilize a  `v-model="isValid" property which will be our local model for determining the validity state of an input or collection of inputs. 
We can also assign a `ref` in order to prompt validations when we choose too, if requirements call for it at a specific time (ie at File and Pay or something)